test/spec/com/cinergix/mapper/objectmapper/GetFieldColumnMappingTest.java

```diff
@@ -0,0 +1,208 @@
+package com.cinergix.mapper.objectmapper;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.Field;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+
+import org.junit.Test;
+
+import com.cinergix.mapper.ObjectMapperTestAbstract;
+import com.cinergix.mapper.data.SimpleColumnMappedUserMock;
+import com.cinergix.mapper.data.SimpleUserMock;
+import com.cinergix.mapper.data.UserMock;
+
+public class GetFieldColumnMappingTest extends ObjectMapperTestAbstract {
+	
+	private ExtendedObjectMapper mapper = new ExtendedObjectMapper();
+	
+	private HashMap<Field, String> invokeCheckColumnLabelExist( Class dataClass, ResultSet result ){
```

Comment:
This function is unecessary. We can do the same thing in the ExtendedObjectMapper.testGetFieldColumnMapping.